docs/user-guides/governance/governance.md

```diff
@@ -64,13 +64,17 @@ In version v1.5 command to initialize governance is available in the Partner Cha
 	--genesis-utxo <GENESIS_UTXO> \
 	--ogmios-url <OGMIOS_URL> \
 	--payment-key-file <PAYMENT_KEY_FILE> \
-	--new-governance-authority <NEW_GOVERNANCE_AUTHORITY>
+	--old-governance-authority <OLD_GOVERNANCE_AUTHORITY> \
```

Comment:
I have a fundamental question: do we need this?

I think the answer is "no". What is more, we add a burden on user, to give us this value properly. This opens new error paths.

Aren't we able get it from the on-chain data? I'm wondering about extending `scripts-data` with optional output of the current multisig or adding a command for getting on chain. Such addition would not be required but could help users.

Reply:
A least the parameter name is wrong. Because it is not old/current governance authorities but expected required signers (sorry, going to use technical name, because I can't find a good domain name for it).

Reply:
What about this? I argue it's not required - see #585